openapi/src/parts/controlled_azure_storage_container.yaml

```diff
@@ -0,0 +1,62 @@
+paths:
+  /api/workspaces/v1/{workspaceId}/resources/controlled/azure/storageContainer:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+    post:
+      summary: Create a new controlled Azure Storage Container
+      operationId: createAzureStorageContainer
+      tags: [ControlledAzureResource]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateControlledAzureStorageContainerRequestBody'
+      responses:
+        '200':
+          description: Response to create controlled azure storage
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CreatedControlledAzureStorageContainer'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+components:
+  schemas:
+    AzureStorageContainerCreationParameters:
+      description: >-
+        Storage container-specific properties to be set on creation. These are a subset of the values
+        accepted by the azure resource API
+      type: object
+      required: [ name, storageAccountName ]
+      properties:
+        name:
```

Comment:
In keeping with other resources, I used `name` here instead of `storageContainerName`. However, I could see going with `storageContainerName` for clarity and to be consistent with what is used for the container attributes.

Thoughts?

Reply:
I'm in favor of the response + request using the same name for parameters; we should adjust the storage account parameters to do this as well.

Reply:
I would like the name to be the same, but it will be a breaking change for folks using the API client. Not a big deal for Rawls because I can easily update the code in my next PR. @ddietterich do you think it's OK to change the name of the request attribute? I'm not sure if anyone (outside of Rawls) is actually using this part of the API yet.

Reply:
I think the API is not used in production and only used in that one place, right? If so, it should be no problem to change both sides. We have made changes like that before.

Reply:
I have changed both the storage account and storage container request attributes to use the same name as what is returned in the creation response.